import-core/import-rest/src/main/java/org/devgateway/importtool/rest/ImportController.java

```diff
@@ -106,28 +119,41 @@
 		request.getSession().removeAttribute(DESTINATION_PROCESSOR);
 		request.getSession().removeAttribute(SESSION_TOKEN);
 		request.getSession().removeAttribute(DOCUMENT_MAPPER);
+		request.getSession().removeAttribute("IATI_STORE_ACTIVITIES");
```

Comment:
@gmutuhu I removed this wipe since the that attribute is populated in the previous step and its needed to generate filters